### DISCUSSION TITLE
My contribution not in profile?

### DISCUSSION THREAD

**Discussion by lequocvuong** [community/community#37425] [Other Feature Feedback, Questions, & Ideas]
My contribution not in profile?  Please check for me if I wrong?

**Comment by radar-blip** [comment]
You have 89 contributions in the last year.

![Screenshot from 2022-10-26 23-59-13](https://user-images.githubusercontent.com/39196767/198188241-bcacf550-ffd5-49d5-be21-900adfc4f33b.png)

**Reply by lequocvuong** [reply]
![Screen Shot 2022-10-27 at 11 14 24](https://user-images.githubusercontent.com/33302910/198189935-1ea04873-df1b-4c5f-be05-536fe98451a3.png)
My contributions only show 88. Today 2022/10/27. I already commit multiple commit but not show in graph.

**Reply by radar-blip** [reply]
You made a commit in a private repository, those wont show unless you enable them (in your settings).

![profiles](https://user-images.githubusercontent.com/39196767/198190556-016eea49-3a6a-493a-811c-adf1f2b1f535.png)

**Reply by lequocvuong** [reply]
I already include private contributions on my profile before.

**Reply by radar-blip** [reply]
Did you also check here: 

![prof](https://user-images.githubusercontent.com/39196767/198192158-184f6ee4-6be4-4b24-8767-b5245a93b248.png)

**Reply by lequocvuong** [reply]
![Screen Shot 2022-10-27 at 11 34 47](https://user-images.githubusercontent.com/33302910/198192303-2824216f-f06e-4f51-bf72-2ecbfb4c9452.png)
I also turn on private contributions on my profile. But my contributions show 88, you check show 89 is difference in here. I also try commit many time today, but not show in my profile.

**Reply by radar-blip** [reply]
I dont know. I think maybe it takes some time to update?

**Comment by DenverCoder1** [comment]
This article might help - https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile

> Commits will appear on your contributions graph if they meet all of the following conditions:
> 
> - The email address used for the commits is associated with your account on GitHub.com.
> - The commits were made in a standalone repository, not a fork.
> - The commits were made:
>   - In the repository's default branch
>   - In the `gh-pages` branch (for repositories with project sites)
> 
> In addition, at least one of the following must be true:
> 
> - You are a collaborator on the repository or are a member of the organization that owns the repository.
> - You have forked the repository.
> - You have opened a pull request or issue in the repository.
> - You have starred the repository.

> After making a commit that meets the requirements to count as a contribution, you may need to wait for up to 24 hours to see the contribution appear on your contributions graph.

You can check the [docs link](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile) for more possible reasons.

**Reply by lequocvuong** [reply]
<img width="1241" alt="Screen Shot 2022-10-28 at 9 11 12 PM" src="https://user-images.githubusercontent.com/33302910/198636908-2469ed91-b471-4d4c-a00f-8e1b46485e77.png">
As you see, I made many commit to master but graph not count. I have owner project, and I commit to main branch. What am I to do now?

**Reply by DenverCoder1** [reply]
Have you made sure the email address associated with the commits is linked to your GitHub account?

That image shows the commits are from `VuongLe`, but your GitHub profile is `lequocvuong` so it seems like it may not be associating the commits with your account.

**Reply by lequocvuong** [reply]
Thanks, I try set username, email again.

**Comment by jfullstackdev** [comment]
I'm seeing those commits not verified, make sure it's verified, there are many instances so I can't pin it down, but my example is what I'm doing,

instead of using GitHub Desktop in my case, because I did not install Git, I will manually create a branch on GitHub and create a PR on GitHub, that way, all of my commits are recognized as mine 

this might help

https://daily-dev-tips.com/posts/how-to-verify-your-commits-on-github/

**Reply by lequocvuong** [reply]
Thanks I try solution i try @DenverCoder1 's solution and it's working now.

**Reply by ajayidhikrullah** [reply]
How did you resolved this please? @lequocvuong
